app/logic/Calendar/ActiveSync/ActiveSyncEvent.ts

```diff
@@ -176,6 +189,15 @@ export class ActiveSyncEvent extends Event {
         throw new ActiveSyncError("Sync", response.Responses.Delete.Status, this.calendar);
       }
     }
+    if (this.response == InvitationResponse.Organizer) {
+      await this.outgoingInvitation.sendCancellations(this.calendar.account);
+    } else if (this.response) {
+      for (let participant of this.participants) {
+        if (participant.response == InvitationResponse.Organizer) {
+          await this.sendInvitationResponse(InvitationResponse.Decline, this.calendar.account);
+        }
+      }
```

Comment:
These lines should be on a class `IncomingInvitation`, but class is created in #550, correct? Can you please do the change there in that PR, once this PR here lands?